pkg/adapterManager/manager.go

```diff
@@ -18,122 +18,193 @@ import (
 	"fmt"
 	"sync"
 
```

Comment:
any idea why there is an extra newline here? it feels like the "bytes"... section should be combined (and sorted) with the "fmt" and "sync" import block.